java/src/org/openqa/selenium/remote/service/DriverService.java

```diff
@@ -456,10 +420,6 @@ public DS build() {
         port = PortProber.findFreePort();
       }
 
-      if (exe == null) {
-        exe = findDefaultExecutable();
-      }
-
```

Comment:
Seems this bit is missing when using the Grid, `DriverServiceSessionFactory` uses the service directly. I need to check that part. Some tests are failing because of that.